spec/models/change_service_event_spec.rb

```diff
@@ -21,6 +21,19 @@
       expect(change_service_event.data['name']).to eql(new_service_name)
     end
   end
+  context '#trigger_publish_event' do
+    it 'when change service event occurs it triggers a publish event' do
```

Comment:
maybe `it 'when entity id is changed'` since we are already in ChangeServiceEvent spec anyway